libraries/AP_GyroFFT/AP_GyroFFT.cpp

```diff
@@ -53,15 +67,15 @@ const AP_Param::GroupInfo AP_GyroFFT::var_info[] = {
     // @Param: MINHZ
     // @DisplayName: Minimum Frequency
     // @Description: Lower bound of FFT frequency detection in Hz.
-    // @Range: 10 400
+    // @Range: 80 400
     // @Units: Hz
     // @User: Advanced
     AP_GROUPINFO("MINHZ", 2, AP_GyroFFT, _fft_min_hz, 80),
 
     // @Param: MAXHZ
     // @DisplayName: Maximum Frequency
     // @Description: Upper bound of FFT frequency detection in Hz.
-    // @Range: 10 400
+    // @Range: 80 400
```

Comment:
same here

Reply:
Ok, I chatted to @lthall about this and we agreed that 20Hz was a reasonable minimum to cover most vehicle sizes. I have changed it to this and updated the docs to reflect the difference between large and small vehicles